src/chainstate/stacks/index/node.rs

```diff
@@ -1490,35 +1504,26 @@ mod test {
         for i in 0..15 {
             assert!(node16.insert(&TriePtr::new(TrieNodeID::Node48, (i+1) as u8, (i+2) as u32)));
         }
-        let node16_bytes = vec![
+        let mut node16_bytes = vec![
             // node ID
             TrieNodeID::Node16,
-            // ptrs (16)
-            TrieNodeID::Node48, 0x01, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x02, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x03, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x04, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x05, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x06, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x07, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x08, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x09, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x0a, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x0b, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x0c, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x0d, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x0e, 0, 0, 0, 0, 
-            TrieNodeID::Node48, 0x0f, 0, 0, 0, 0, 
-            TrieNodeID::Empty, 0x00, 0, 0, 0, 0, 
+        ];
```

Comment:
Same here -- please use a raw array of bytes.

Reply:
Addressed in b8594a21